invenio_notifications/manager.py

```diff
@@ -29,27 +29,7 @@ def _backend_exists(self, backend_id):
         """Check if backend is registered."""
         return backend_id in self._backends
 
-    def _dispatch_notification(self, notification, backend, **kwargs):
-        """Extend notification and start task for sending."""
-        try:
-            extended_notification = backend.extend_notification(
-                notification.copy(), **kwargs
-            )
-
-            backend.send_notification.apply_async(
-                args=[extended_notification.dumps()],
-            )
-
-        except Exception as e:
-            current_app.logger.warning(e)
-            raise e
-
-    def broadcast(self, notification, **kwargs):
-        """Broadcast notification to backends."""
-        for backend in self._backends.values():
-            self._dispatch_notification(notification, backend, **kwargs)
-
-    def notify(self, notification, backend_id, **kwargs):
+    def notify_backend(self, backend_notification, backend_id, **kwargs):
```

Comment:
Very minor: I am allergic to `manager` and similar names if possible :)
What about call this object simply `Notifications`?

You can of course keep it like this, this very non-important :)

Reply:
I'd like to keep the `manager` here, as it handles (de)registering of backends as well (might be more functionality in the future). `Notifications` alone would not give enough context, imo :)